picas/executers.py

```diff
@@ -16,13 +16,14 @@ def execute(args, shell=False):
     @param args: the arguments as they need to be specified for Popen.
     @return: a tuple containing the exitcode, stdout & stderr
     """
-    with Popen(args, stdout=PIPE, stderr=PIPE, shell=shell) as proc:
-        (stdout, stderr) = proc.communicate()
-    return (proc.returncode, stdout, stderr)
+    proc = Popen(args, stdout=PIPE, stderr=PIPE, shell=shell)
```

Comment:
Why is the "with" statement removed? Better for exception handling and resource management with 'with' statement.

Reply:
You are right, seems like a mistake on my part